Update icinga.adoc (#735)
SUSE/doc-susemanager: modules/administration/pages/icinga.adoc

systemctl enable postgresql.service
systemctl start postgresql.service
----

> . Become postgres system user:

Reviewer: ```suggestion
. Become the ``postgres`` system user:
```

----
sudo su - postgres
----
. Start a SQL CLI:
----
psql
----

> . Then setup a new password for the administrative user postgres, and configure a new icinga user:

Reviewer: ```suggestion
. Create a new password for the administrative user ``postgres``, and configure a new ``icinga`` user:
```

----
ALTER USER postgres WITH PASSWORD '<newpassword>';
CREATE USER icinga;
ALTER USER icinga WITH PASSWORD 'icinga';
CREATE DATABASE icinga;
GRANT ALL ON DATABASE icinga TO icinga;
\q
----

> Finally, exit the postgres system user session:

Reviewer: ```suggestion
Exit the ``postgres`` system user session:
```

systemctl reload postgresql.service
----

> . Configure the database schema by running the following command:

Reviewer: ```suggestion
. Configure the database schema with this command:
```

[IMPORTANT]
.Open Firewall Port
====

> Allow port `80` through your firewall or you will not be able to access the WebGUI.

Author: Either I am quite wrong, or 5432 was incorrect. 5432 is for PostgreSQL, that it's to be used by ido2b when it receives data from nrpe.

Reviewer 2: That is correct, thank you.

> Allow port `80` through your firewall or you will not be able to access the WebGUI.

Reviewer: ```suggestion
Allow port `80` through your firewall or you will not be able to access the {webui}.
```

systemctl start apache2.service
----

> . Login to the WebGUI at: http://<client-hostname>/icinga.

Reviewer: ```suggestion
. Log in to the {webui} at: http://<client-hostname>/icinga.
```

systemctl start apache2.service
----

> . Login to the WebGUI at: http://<client-hostname>/icinga.

Author: If we say 'localhost', the user needs to use a browser on the very same client. I think it makes more sense to use the hostname for the client.

Reviewer 2: This should probably just be *hostname*

> This concludes setup and initial configuration of Icinga.

Author: @Loquacity I am worried about the next section. It's about monitoring the server with Icinga, but AFAIK we don't support that anymore (we have prometheus for that).

Do we want to invest time on rewriting the whole section to describe how to monitor a simple version and then just point to the official doc?

Reviewer: I wouldn't bother personally. It seems pretty unlikely that this will be published any time soon. Just comment it out.

Reviewer 2: @juliogonzalez I agree with lana, lets comment this section out for now. We are promoting prometheus.
